concourse/pipeline.yml

```diff
@@ -30,6 +37,18 @@ resources:
         - concourse/Dockerfile
         - Gemfile*
 
+  - name: grafana-annotate-deploy
+    type: grafana-annotation
+    icon: chart-areaspline
+    source:
+      url: https://grafana-paas.cloudapps.digital
+      api_token: ((grafana-api-key))
+      tags:
```

Comment:
I've not done much with grafana annotation and tags before.
I was surprised to see these not targeted at a dashboard but rather overall...

So I'm thinking we use the query annotation feature to then target those to our dashboards.
I'd figured therefore we should make these:
- one that idenifies our team
- one that identifies our app
- one that identifies the event
- one that identifies the env

Then a start / stop in the params?
Everyone agree that works?